### ISSUE TITLE
Doesn't this override all the grided predictions?

### ISSUE THREAD

**Issue by AndreisBruno** [hszhao/semseg#24]
Doesnt this override predictions?
https://github.com/hszhao/semseg/blob/dae0274fd75759530ce972160ddf0ecc561adbea/tool/demo.py#L181

**Comment by hszhao** [comment]
We do summation/average over the logits on grid regions.
https://github.com/hszhao/semseg/blob/dae0274fd75759530ce972160ddf0ecc561adbea/tool/demo.py#L159
